.github/workflows/pull-request.yml

```
@@ -20,479 +9,53 @@ env:
-          fi     
-      
-      - name: TypeScript compilation for changed files
-        run: |
-          for file in ${{ steps.changed-files.outputs.all_files }}; do
-            if [[ "$file" == *.ts || "$file" == *.tsx ]]; then
-              npx tsc --noEmit "$file"
+          while [ $TIME_ELAPSED -lt $MAX_WAIT_TIME ]; do
+            echo "Fetching PR reviews from GitHub API..."
+            reviews=$(curl -s -H "Authorization: token ${{ secrets.GITHUB_TOKEN }}" \
+              "https://api.github.com/repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews")
+            
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)